mythtv/libs/libmythservicecontracts/services/dvrServices.h

```diff
@@ -45,11 +45,12 @@
 class SERVICE_PUBLIC DvrServices : public Service  //, public QScriptable ???
 {
     Q_OBJECT
-    Q_CLASSINFO( "version"    , "6.1" );
+    Q_CLASSINFO( "version"    , "7.0" );
```

Comment:
Mitch,

I got the version wrong, it should be 6.2 (stuartm answered my question on IRC)